content/docs/lists-and-keys.md

```diff
@@ -119,34 +118,34 @@ const todoItems = todos.map((todo) =>
 );
 ```
  
-When you don't have stable IDs for rendered items, you may use the item index as a key as a last resort:
+Kiedy nie będziesz miał stabilnych identyfikatorów dla wyświetlanych elementów, w skrajnym przypadku możesz użyć indeksu jako klucz:
  
 ```js{2,3}
 const todoItems = todos.map((todo, index) =>
-  // Only do this if items have no stable IDs
+  // Rób to tylko w przypadku gdy elementy nie mają stabilnego identyfikatorów
   <li key={index}>
     {todo.text}
   </li>
 );
 ```
  
-We don't recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny's article for an [in-depth explanation on the negative impacts of using an index as a key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.
+Nie zalecamy używania indeksów jako klucze jeżeli kolejność elementów może ulec zmianie. Może to negatywnie wpłynąć na wydajność i spowodować problemy ze stanem komponentu. Sprawdź artykuł Robin Pokorny na temat [szczegółowego wyjaśnienia negatywnych aspektów używania indeksu jako klucz](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). Jeżeli nie zdecydujesz się jawnie ustawić kluczy dla elementów listy, React domyślnie użyje indeksów jako klucze.
  
-Here is an [in-depth explanation about why keys are necessary](/docs/reconciliation.html#recursing-on-children) if you're interested in learning more.
+Jeżeli chciałbyś dowiedzieć się wiecej, to tutaj znajdziesz [szczegółowe wyjaśnienie dlaczego klucze są niezbędne](/docs/reconciliation.html#recursing-on-children).
  
-### Extracting Components with Keys {#extracting-components-with-keys}
+### Wydrębnianie komponentów z kluczami {#extracting-components-with-keys}
```

Comment:
```suggestion
### Wyodrębnianie komponentów z kluczami {#extracting-components-with-keys}
```
lub:
```suggestion
### Wyciąganie komponentów z kluczami {#extracting-components-with-keys}
```
bo tak zostało to przetłumaczone na stronie o komponentach: https://github.com/reactjs/pl.reactjs.org/pull/45/files#diff-becb6cc71f7755db88023cfd07ca9a1fR125

Osobiście wolę "wyodrębnianie" i zmieniłbym w obu miejscach.

Reply:
@jakubdrozdek Nie orientuje się jak wyglądała tam sprawa z `Wyciąganie` vs `Wyodrębnianie` ?
Osobiście jestem za @jfrej i wybrałbym `Wyodrębnianie` oraz naniósł poprawkę pod tym lub osobnym PR do https://github.com/reactjs/pl.reactjs.org/pull/45/files#diff-becb6cc71f7755db88023cfd07ca9a1fR125.

Daj znać co o tym sądzisz.

Reply:
Masz rację. Ja się zajmę poprawką w tamtym pliku. Tutaj niech zostaną rzeczy związane z Listami i kluczami.

Reply:
Poprawki wrzucone do osobnego PR-a: #70